src/Task/ApiGen/ApiGen.php

```diff
@@ -30,6 +31,7 @@ class ApiGen extends BaseTask implements CommandInterface
      * @var string
      */
     protected $command;
+    protected $operation = ' generate';
```

Comment:
Yeah, I was just looking at the rest of the code and realizing that the way I want the code to be, and the way it currently is, is not always the same.